samples/tfm_integration/tfm_ipc/README.rst

```diff
@@ -103,6 +103,35 @@ Or, post build:
 
       $ ninja run
 
+On ST Nucleo L552ZE Q:
+======================
+
+Build Zephyr with a non-secure configuration:
+
+   .. code-block:: bash
+
+      $ west build -b nucleo_l552ze_q_non_secure samples/tfm_integration/tfm_ipc/
+
+Three scripts are generated in the ``build/tfm/install`` folder:
+
+  - ``postbuild.sh``: Updates regression.sh and TFM_UPDATE.sh scripts according to generated flash_layout.h.
+  - ``regression.sh``: Sets platform option bytes config and erase platform.
+  - ``TFM_UPDATE.sh``: Writes bl2, secure, and non secure image in target.
+
+Run them in the following order to flash the board:
+
+   .. code-block:: bash
+
+      $ ./build/tfm/install/postbuild.sh
+      $ ./build/tfm/install/regression.sh
+      $ ./build/tfm/install/TFM_UPDATE.sh
+
+Reset the board.
+
+ .. note::
+      Note that and arm-none-eabi-gcc should be available (defined in PATH) and that STM32_Programmer_CLI is required to run ``regression.sh`` and ``TFM_UPDATE.sh``. (see https://www.st.com/en/development-tools/stm32cubeprog.html)
```

Comment:
Word missing between `and` and `arm-none-eabi-gcc`?
```suggestion
      Note that ``arm-none-eabi-gcc`` should be available in the PATH variable, and that ``STM32_Programmer_CLI`` is required to run ``regression.sh`` and ``TFM_UPDATE.sh``. (see https://www.st.com/en/development-tools/stm32cubeprog.html)
```

Reply:
What about cases where the Zephyr SDK is used on Linux (`arm-zephyr-gcc` instead of `arm-none-eabi-gcc`)? Does the script handle those as well?

Reply:
I have updated the code to add all the suggested changes.